templates/content/blank/NewApp.iOS/NewApp.iOS.fsproj

```diff
@@ -190,5 +190,4 @@
       <Name>NewApp</Name>
     </ProjectReference>
   </ItemGroup>
-  <Import Project="../.nuget/NuGet.targets" />
```

Comment:
Not sure why this is needed - causes builds of the created project to fail as it tried to find NuGet.exe in the project